lightningd/options.c

```diff
@@ -398,6 +398,12 @@ static void config_register_opts(struct lightningd *ld)
 			 opt_set_u64, opt_show_u64,
 			 &ld->ini_autocleaninvoice_cycle,
 			 "If expired invoice autoclean enabled, invoices that have expired for at least this given seconds are cleaned");
+	opt_register_arg("--max-funding-unconfirmed-blocks",
+			 opt_set_u32, opt_show_u32,
+			 &ld->max_funding_unconfirmed,
+			 "Maximum number of blocks we wait for a channel "
+			 "funding transaction to confirm, if we are the "
+			 "fundee.");
 }
```

Comment:
I've been removing options like this in my documentation patch; I think this should just be a constant in a header somewhere.  I can't see why anyone would want to change it, unless we want to change it for everyone.

Reply:
I kept it since it is useful in the test to lower the threshold.  Or should I make it into a `dev-` option instead?

Reply:
Making this a `dev-` option would probably be sensible, yes :-)